tox.ini

```diff
@@ -55,6 +55,7 @@ passenv =
 deps =
   dnspython
   pytest
+  PyYAML
```

Comment:
setup.py has a very serious flaw where we cannot declare dependencies whose pypi name is significantly different from its module import name.

See: https://mesosphere.atlassian.net/browse/DCOS-11783

If you just add PyYaml to setup, then when using any of the entrypoints, you will be greeted with a an exception saying "could not import module 'pyyaml'"